_example/_markdown.md

```diff
@@ -2,28 +2,28 @@
 
 Example command using cobra and cobradoc.
 
-```
+```text
```

Comment:
Many Markdown linters - like the kind that my documentation has to go through - will flag these under [`markdownlint/MD040`](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md040.md), which requires fenced code blocks like this to have a language specified. I came to the conclusion that plain text describes the contents best.

Since fenced code blocks are used so much in the generated documentation, I would prefer the tool to take care of that for us instead of having to add them manually.

Reply:
Ah, linters, I see.